programs/mmm/src/state.rs

```diff
@@ -10,6 +10,7 @@ pub const ALLOWLIST_KIND_FVCA: u8 = 1;
 pub const ALLOWLIST_KIND_MINT: u8 = 2;
 pub const ALLOWLIST_KIND_MCC: u8 = 3;
 pub const ALLOWLIST_KIND_METADATA: u8 = 4;
+pub const ALLOWLIST_KIND_ANY: u8 = u8::MAX;
```

Comment:
Should we mention that checks should be done by co-signer?

Reply:
yeah it's implied, can add as a comment